cumulus/client/cirrus-executor/src/tests.rs

```diff
@@ -40,7 +40,6 @@ async fn test_executor_full_node_catching_up() {
 
 	// run cirrus dave (a secondary chain full node)
 	let dave = cirrus_test_service::TestNodeBuilder::new(tokio_handle, Dave)
-		.connect_to_parachain_node(&charlie)
```

Comment:
Why was this a problem?

Reply:
According to the log https://github.com/subspace/subspace/runs/6441917635?check_suite_focus=true, the node `//Dave (parachain)` somehow resets to the block#0 after a few blocks(#5) and is no longer able to move forward, I guess after the syncing issue was fixed, the two ways of producing new execution chain blocks conflicts in some place. But I think it's unnecessary to dig into it since we don't need the traditional syncing anyway. BTW, I have never seen that locally, whether running the tests locally or running a local network.

Reply:
We don't need it, but we do have it and will have for a while, so I feel like it just masks an actual problem in tests, but we'll hit it in actual deployment sooner or later anyway.

Reproduction locally might be easier when machine is under load, try to run tests while running something heavy in parallel, like OpenSSL benchmark (`openssl speed -multi 24`).

Reply:
Haven't found anything new running tests with a busy machine locally.

You won't have this problem as long as you don't connect to the other executors, only syncing from other executors is affected, which is broken already even worse before this PR.

Unless you want to support the traditional syncing between the executors officially, I don't think we need to invest more time debugging the issue, we already have enough for the final goal.

Reply:
The initial motivation for this PR is that we spent quite some time debugging the syncing issue between the executors as we don't know what's the root cause and if it could affect the other things. Since we figured it out and the fix is simple, I made this PR, without foreseeing the test issue as I have never seen it before. But if it requires more time to fix it properly, it's not profitable if it will not be supported officially IMHO. Hope this explanation can solve some of your confusion :D

Reply:
As I said, things like this might shoot later and it'd be really nice to know what the root cause is regardless.

Reply:
Less likely if this piece of code will not be enabled in practice. I can make it a backlog item and try debugging when our schedule is not pretty tight.

Reply:
I think that is a good idea